### ISSUE TITLE
Blank screen after install

### ISSUE THREAD

**Issue by hamzab70** [invoiceninja/dockerfiles#670]
**Setup information**
docker-compose

**Describe the bug**
Blank screen after install and this reponse for the index.php/index.html file:

```
<!DOCTYPE html>
<html lang="en" class="h-full">
  <head>
    <title>Invoice Ninja</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta
      name="description"
      content="Leading free invoice generator for freelancers and small businesses. Invoice clients, accept payments, track expenses &amp; time billable-tasks online."
    />
    <link rel="stylesheet" href="/rsms/inter.css" />
    <link rel="icon" href="/favicon.ico" />
    <link rel="apple-touch-icon" href="/logo192.png" />
    <link rel="manifest" href="/manifest.json" />
    <script type="module" crossorigin src="/bundle.edcb883a.js"></script>
    <link rel="stylesheet" href="/index-c3763b73.css">
  </head>
  <body class="h-full">
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>    
  </body>
</html>
```

**To reproduce**
Add Cloudlared container to docker-compose.yaml (to use cloudflare tunnel to access the app)
Change ngix port to `8012:80`
Run `docker-compose up -d` (as normal)

**Expected behavior**
Logging screen for invoiceninja.

**Screenshots/logs**
![image](https://github.com/user-attachments/assets/68e53d62-7644-41f1-88a2-b84f04c5e4b8)
![image](https://github.com/user-attachments/assets/04bbff91-6815-4220-bb33-c1b069422dac)

There seems to be no problem in the logs of both the app and nginx containers.
```
app-1                     | 172.20.0.5 -  04/Dec/2024:22:54:48 +0000 "GET /index.php" 200
nginx-1                   | 172.20.0.6 - - [04/Dec/2024:22:54:48 +0000] "GET /bundle.edcb883a.js HTTP/1.1" 200 537 "https://facturas.XYZ.com/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36" "2a09:bac1:23a0::28:104"
app-1                     | 172.20.0.5 -  04/Dec/2024:22:54:48 +0000 "GET /index.php" 200
nginx-1                   | 172.20.0.6 - - [04/Dec/2024:22:54:48 +0000] "GET /index-c3763b73.css HTTP/1.1" 200 537 "https://facturas.XYZ.com/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36" "2a09:bac1:23a0::28:104"
app-1                     | 172.20.0.5 -  04/Dec/2024:22:54:48 +0000 "GET /index.php" 200
nginx-1                   | 172.20.0.6 - - [04/Dec/2024:22:54:48 +0000] "GET /rsms/inter.css HTTP/1.1" 200 537 "https://facturas.XYZ.com/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36" "2a09:bac1:23a0::28:104"
app-1                     | 172.20.0.5 -  04/Dec/2024:22:54:48 +0000 "GET /index.php" 200
```

**Docker/Kubernetes/Helm**:
- Output of `docker version`:
```
➜  debian git:(debian) ✗ docker version
Client: Docker Engine - Community
 Version:           27.3.1
 API version:       1.47
 Go version:        go1.22.7
 Git commit:        ce12230
 Built:             Fri Sep 20 11:41:11 2024
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          27.3.1
  API version:      1.47 (minimum version 1.24)
  Go version:       go1.22.7
  Git commit:       41ca978
  Built:            Fri Sep 20 11:41:11 2024
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.7.23
  GitCommit:        57f17b0a6295a39009d861b89e3b3b87b005ca27
 runc:
  Version:          1.1.14
  GitCommit:        v1.1.14-0-g2c9f560
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```
**Additional context**
Say I stated at the start I'm using Cloudflare Tunnels for secure access but this should be totally transparent for InvoiceNinja as the requests are proxied directly to the nginx container as normal requests. I have other apps with React frontends like Vaulwarden in the same machine being exposed with other tunnels that work without errors.

**Comment by turbo124** [comment]
Does it work without adding in the cloudflare tunnel. those brower console errors are pretty typical of a reverse proxying issue.

**Comment by hamzab70** [comment]
Hello David first of all thank you for the quick answer. Unfortunately after trying with both the Hetzner VM ip:8012 from the browser and with a curl command the output is the same.
```
❯curl 136.243.163.XYZ:8012
<!DOCTYPE html>
<html lang="en" class="h-full">
  <head>
    <title>Invoice Ninja</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta
      name="description"
      content="Leading free invoice generator for freelancers and small businesses. Invoice clients, accept payments, track expenses &amp; time billable-tasks online."
    />
    <link rel="stylesheet" href="/rsms/inter.css" />
    <link rel="icon" href="/favicon.ico" />
    <link rel="apple-touch-icon" href="/logo192.png" />
    <link rel="manifest" href="/manifest.json" />
    <script type="module" crossorigin src="/bundle.edcb883a.js"></script>
    <link rel="stylesheet" href="/index-c3763b73.css">
  </head>
  <body class="h-full">
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>
  </body>
</html>
```
Docker-compose logs:
```
nginx-1                   |  - - [04/Dec/2024:23:16:55 +0000] "GET / HTTP/1.1" 200 957 "-" "curl/7.76.1" "-"
app-1                     | 172.20.0.5 -  04/Dec/2024:23:16:55 +0000 "GET /index.php" 200
nginx-1                   |  - - [04/Dec/2024:23:16:56 +0000] "GET / HTTP/1.1" 200 957 "-" "curl/7.76.1" "-"
app-1                     | 172.20.0.5 -  04/Dec/2024:23:16:56 +0000 "GET /index.php" 200
nginx-1                   | - - [04/Dec/2024:23:18:14 +0000] "GET /rsms/inter.css HTTP/1.1" 200 537 "79.159.XYZ.XYZ" "Mozilla/5.0 (iPhone; CPU iPhone OS 13_2_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.3 Mobile/15E148 Safari/604.1" "-"
app-1                     | 172.20.0.5 -  04/Dec/2024:23:18:14 +0000 "GET /index.php" 200
nginx-1                   | - - [04/Dec/2024:23:18:14 +0000] "GET /index-c3763b73.css HTTP/1.1" 200 537 "" "Mozilla/5.0 (iPhone; CPU iPhone OS 13_2_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.3 Mobile/15E148 Safari/604.1" "-"
nginx-1                   | - - [04/Dec/2024:23:18:14 +0000] "GET /bundle.edcb883a.js HTTP/1.1" 200 537 "/" "Mozilla/5.0 (iPhone; CPU iPhone OS 13_2_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.3 Mobile/15E148 Safari/604.1" "-"
app-1                     | 172.20.0.5 -  04/Dec/2024:23:18:14 +0000 "GET /index.php" 200
app-1                     | 172.20.0.5 -  04/Dec/2024:23:18:14 +0000 "GET /index.php" 200
```

**Comment by turbo124** [comment]
try and curl directly to the .js file 

/bundle.edcb883a.js

are you able to pull it back, or does it return blank?

If you run

```
docker-compose exec nginx ls -la /var/www/html/public/
```

Do you see the specified file?

Also just to note, you may want to update to the latest commit of the debian branch as there have been some important updates around this.

**Comment by hamzab70** [comment]
Hello again David, yes found the problem. 

I was running an old version it seems to be, so the init.sh was not cleaning and copiyng the files as it should. I discovered it checking the nginx container files and playing around with the latest version. 

As is a fresh install I just did a `docker down -v` and pulled the latest version, after executing everything is now working as expected.

Thank you for the quick answers and support!
